app/models/user.rb

```diff
@@ -39,6 +39,16 @@ class User < ApplicationRecord
                 90.days.ago)
         }
 
+  scope :not_active_for,
+        lambda { |days|
```

Comment:
The SQL scope could be simplified by passing the method an instance of `Date` rather than time (i.e. `76.days.ago.to_date`) and using MySQL's `DATE`:
```ruby
where('DATE(current_login_at) = ? OR
  (current_login_at IS NULL AND DATE(created_at) = ?)', date, date)
```